uint64 timestamp
Position pos
uint32 charge
uint32 load
uint32 route_length
string last_action
string last_action_result
Item[] items
#Maybe add inFacility and fPosition if necessary
#More optional information like current route, last action, last action result could be added as well